screener.config.js

```diff
@@ -0,0 +1,46 @@
+const _ = require('lodash')
+const glob = require('glob')
+const path = require('path')
+
+// https://github.com/screener-io/screener-runner
+const screenerConfig = {
+  projectRepo: 'stardust-ui/react',
+
+  apiKey: process.env.SCREENER_API_KEY,
```

Comment:
ideally we might think about introducing dedicated script (or augment test:visual script) with logic that will perform all the necessary configuration on the developer's machine

Reply:
Agree on the principles proposed.

In the case of image diff comparisons, I'm not sure this is something we should allow locally.  Different platforms (mac, win, linux) render pixels inconsistently and fail image diffs.